ipatests/test_integration/test_commands.py

```diff
@@ -679,3 +679,31 @@ def test_user_mod_change_capitalization_issue5879(self):
         assert 'Modified user "%s"' % (ipauser) in cmd.stdout_text
         assert 'First name: %s' % (modfirst) in cmd.stdout_text
         assert 'Last name: %s' % (modlast) in cmd.stdout_text
+
+    def test_ipa_adtrust_install_with_locale_issue8066(self):
+        """
+        This test checks that ipa-adtrust-install command runs successfully
+        on a system with locale en_IN.UTF-8 without displaying error below
+        'IndexError: list index out of range'
+
+        This is a testcase for Pagure issue
+        https://pagure.io/freeipa/issue/8066
+
+        """
+        # Set locale to en_IN.UTF-8
+        tasks.kinit_admin(self.master)
+        self.master.run_command(['localectl', 'set-locale',
+                                 'LANG=en_IN.UTF-8'])
+        result = self.master.run_command(['localectl', 'status'])
+        assert 'System Locale: LANG=en_IN.UTF-8' in result.stdout_text
+        # Install ipa-server-adtrust and check status
+        msg1 = 'Unexpected error - see /var/log/ipaserver-install.log' \
+               'for details'
+        msg2 = 'IndexError: list index out of range'
+        result = self.master.run_command(['dnf', 'install', '-y',
+                                          'freeipa-server-trust-ad'])
+        assert result.returncode == 0
+        result = self.master.run_command(['ipa-adtrust-install', '-U'])
+        assert result.returncode == 0
+        assert msg1 not in result.stdout_text
```

Comment:
I'm not sure the text asserts are needed since you are checking that the returncode of the installer is 0 indicating success. I guess it doesn't hurt anything but just seems extraneous.